Automattic/vip-go-mu-plugins
# Revert cron-control-next to what's running in production

## Description

The current version of `cron-control-next` isn't in use and we need to test out a different fix, so this wipes the slate clean for that testing.

`cron-control-next` should now match `cron-control`.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Add `define( 'VIP_CRON_CONTROL_USE_NEXT_VERSION', true );` to `vip-config/vip-config.php`
1. Verify normal site functionality
1. Verify cron functionality with CLI commands - `wp cron-control events list`, `wp cron event run --due-now`, etc
